tests/test_couch_migration.py

```diff
@@ -1,15 +1,15 @@
 from __future__ import print_function
-
 from __future__ import absolute_import
 from __future__ import unicode_literals
+from io import open
 import os
 import shutil
 
-import yaml
 from couchdb_cluster_admin.doc_models import ShardAllocationDoc
 from mock.mock import patch
 from nose.tools import assert_equal
 from parameterized import parameterized
+import yaml
```

Comment:
Conventionally bare `import ...`s go above `from ... import ...`

See also [isort](https://pypi.org/project/isort/) to automate import sorting along with https://github.com/dimagi/commcare-cloud/pull/4325

Reply:
Pushed changes after running isort on these files. Thanks.